README.md

```diff
@@ -16,19 +16,19 @@
 
 <img src="https://media.giphy.com/media/10l79ICohTu4iQ/giphy.gif" align="right" alt="Unlocking Door">
 
-**Homebridge** is a lightweight Node.js server you can run on your home network that emulates the iOS HomeKit API. It supports Plugins, which are community-contributed modules that provide a basic bridge from HomeKit to various 3rd-party APIs provided by manufacturers of "smart home" devices. 
+**Homebridge** is a lightweight Node.js server you can run on your home network that emulates the iOS HomeKit API. It supports Plugins, which are community-contributed modules that provide a basic bridge from HomeKit to various 3rd-party APIs provided by manufacturers of "smart home" devices.
```

Comment:
That’s actually incorrect.  It emulates the HomeKit Accessory Protocol.

Reply:
This is doubly untrue because my home network does not emulate either HomeKit Accessory Protocol or iOS HomeKit API 😋😋😋 (it's about sentence order 😉).

Reply:
I think you also have a point, but I'd better leave the sentence order to the native speakers ;-)

Reply:
updated to:

> **Homebridge** is a lightweight Node.js server you can run on your home network to emulate the HomeKit Accessory Protocol (HAP).

Reply:
Perfect! 👍